mne/utils/config.py

```diff
@@ -605,6 +605,47 @@ def _get_gpu_info():
     return out
 
 
+def _get_total_memory():
+    """Return the total memory of the system in bytes."""
+    if platform.system() == "Windows":
+        o = subprocess.check_output(
+            [
+                "powershell.exe",
+                "(Get-CimInstance Win32_ComputerSystem).TotalPhysicalMemory",
+            ]
+        ).decode()
+        total_memory = int(o)
+    elif platform.system() == "Linux":
+        o = subprocess.check_output(["free", "-b"]).decode()
+        total_memory = int(o.splitlines()[1].split()[1])
+    elif platform.system() == "Darwin":
+        o = subprocess.check_output(["sysctl", "hw.memsize"]).decode()
+        total_memory = int(o.split(":")[1].strip())
+    else:
+        total_memory = -1
```

Comment:
why not `raise` here, so that later the exception will get caught and a `?` will get displayed?

Reply:
Good idea, done!